### PR TITLE
BatchCom Title Bar display

### LINKED ISSUE #1243: BatchCom Tab not displaying
Tab title not being shown. 
Click on **Miscellaneous > BatchCom**

kazam_screenshot_00041

### PR DESCRIPTION
**Pull request by muarachmann**
This fixes #1243 @teryhill , @aethelwulffe  have a look.

### CODE REVIEW

File: interface/usergroup/user_info.php [hunk 1]
```diff
@@ -12,32 +12,67 @@
...
     <?php
-      call_required_libraries(['bootstrap', 'jquery-min-1-9-1']);
+      call_required_libraries(array("jquery-min-3-1-1","bootstrap","font-awesome", "iziModalToast"));
       ?>
     <script language='JavaScript'>
       //Validating password and display message if password field is empty - starts
       var webroot='<?php echo $webroot?>';
       function update_password()
       {
-          top.restoreSession();
```
Comment (tmccormi) [inline interface/usergroup/user_info.php]: Same here, I believe top.restoreSession() is required.

File: interface/usergroup/user_info_ajax.php [hunk 2]
```diff
@@ -28,33 +28,43 @@
...
-include_once("../globals.php");
-require_once("$srcdir/authentication/password_change.php");
+    include_once("../globals.php");
+    require_once("$srcdir/authentication/password_change.php");
 
-$curPass=$_REQUEST['curPass'];
-$newPass=$_REQUEST['newPass'];
-$newPass2=$_REQUEST['newPass2'];
+    $curPass=$_REQUEST['curPass'];
+    $newPass=$_REQUEST['newPass'];
```
Comment (tmccormi) [inline interface/usergroup/user_info_ajax.php]: THis file should not be in the commit at all.

File: interface/usergroup/user_info.php [hunk 3]
```diff
@@ -12,32 +12,67 @@
     <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
```
Comment (tmccormi) [inline interface/usergroup/user_info.php]: This file should not be in this commit

**Comment by muarachmann** [comment]
Sorry yeah have to rebase my master

**Pull request closed without merging**